Makefile

```diff
@@ -78,6 +78,14 @@ endif
 install.finch-core-dependencies:
 	OUTDIR=$(OUTDIR) ARCH=$(ARCH) "$(MAKE)" -C $(FINCH_CORE_DIR) install.dependencies
 
+# For Finch on macOS and Windows, the container runtime archive locations and digests are set
+# based on the values set in deps/finch-core/deps/container-runtime-full-archive.conf
+-include $(FINCH_CORE_DIR)/deps/container-runtime-full-archive.conf
+CONTAINER_RUNTIME_ARCHIVE_AARCH64_LOCATION := "$(ARTIFACT_BASE_URL)/$(AARCH64_ARTIFACT)"
```

Comment:
I think there's value in using `?=` for these expressions, in case in dev we need to update them to e.g. test against a new nerdctl version

Reply:
Maybe while I have the hood open on this I will go ahead and grab FINCH_IMAGE_LOCATION and FINCH_IMAGE_DIGEST for both platforms as well.